pages/index.js

```diff
@@ -1,12 +1,17 @@
-import SpotList from "../components/SpotList.js";
+import Spots from "../pages/spots/index.js";
 import Footer from "../components/Footer.js";
 import Header from "../components/Header.js";
+import { useState } from "react";
+import { mockSpots } from "../helpers/mockSpots.js";
+
 export default function Home() {
+  const [spots, setSpots] = useState(mockSpots);
```

Comment:
you can remove this an instead rely on the props passed down to the generic Component in _app.js.
```suggestion
export default function Home({ spots, toggleBookmark }) {
```